.github/dependabot.yml

```diff
@@ -1,17 +1,5 @@
 version: 2
 updates:
-  - package-ecosystem: "pip"
-    directory: "./"
-    schedule:
-      interval: "weekly"
-  - package-ecosystem: "pip"
-    directory: "image_generation/stable_diffusion_1_5/cpp/scripts/"
-    schedule:
-      interval: "weekly"
-  - package-ecosystem: "pip"
-    directory: "image_generation/lcm_dreamshaper_v7/cpp/scripts/"
-    schedule:
-      interval: "weekly"
```

Comment:
but it looks like we need to add `tools` with WWB & LLM bench

Reply:
@peterchen-intel, are you OK if I add WWB & LLM bench?

Reply:
@eaidova @AlexKoff88